### ISSUE TITLE
Add tagging to AWS::MediaConnect::Flow resources via CloudFormation

### ISSUE THREAD

**Issue by scotliu** [aws-cloudformation/cloudformation-coverage-roadmap#1415]
### Name of the resource

AWS::MediaConnect::Flow

### Description

Currently AWS::MediaConnect::Flow resources cannot be tagged via CloudFormation[1].

The MediaConnect resources can be done via the console directly or via CLI commands:
https://awscli.amazonaws.com/v2/documentation/api/latest/reference/mediaconnect/tag-resource.html

Documentation on managing Flow tags directly in console here:
https://docs.aws.amazon.com/mediaconnect/latest/ug/flows-manage-tags.html

Requesting review and adding functionality to manage tags on MediaConnect resources via CloudFormation stack/template.  The above also applies to all 5 currently support MediaConnect resources:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/AWS_MediaConnect.html

### Other Details

References
=============
[1] https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-mediaconnect-flow.html

**Comment by flvlef** [comment]
This feature would standardize what's already possible with the other media-related services.

**Comment by timbgn** [comment]
Yep really feel this is missing.  Because it is missing here it is also missing form the terraform awscc provider.
